requestid/requestid.go

```diff
@@ -14,7 +14,8 @@ import (
 // DefaultRequestIDKey is the metadata key name for request ID
 const (
 	DeprecatedRequestIDKey = "Request-Id"
-	DefaultRequestIDKey    = "request_id"
+	DefaultRequestIDKey    = "X-Request-ID"
```

Comment:
Note that this is already implemented in the ingress-controller:
https://github.com/Infoblox-CTO/athena.devops/blob/d9556173c9d749f86515876cadeb6fa0ec9a60d8/cluster-provisioning/charts/base/files/nginx.tmpl#L322-L329

But not forwarded to identity:
https://github.com/Infoblox-CTO/athena.devops/blob/d9556173c9d749f86515876cadeb6fa0ec9a60d8/cluster-provisioning/charts/base/files/nginx.tmpl#L920
(on auth requests, it's set as `Request-ID` only)

Reply:
@kd7lxl, please avoid references to data from private repos in public repositories

Reply:
@kd7lxl We probably can forward it, it a good point

Reply:
broke authz middleware as well https://github.com/Infoblox-CTO/ngp.authz.middleware/pull/23/files#diff-320a504f903c9c37cd1c4852412110f8R216